apps/codeforafrica/src/components/Articles/Articles.js

```diff
@@ -35,7 +35,7 @@ const Articles = React.forwardRef(function Articles(props, ref) {
   const [q, setQ] = useState();
   const [filtering, setFiltering] = useState(false);
   const [tag, setTag] = useState(allTag);
-  const queryParams = useFilterQuery({ page, q, tag: tag.slug });
+  const queryParams = useFilterQuery({ page, q, tag });
```

Comment:
Why the change from `tag.slug` to just `tag`?

Reply:
@kilemensi #610 [updated the `useFilterQuery`](https://github.com/CodeForAfrica/ui/blob/e3eb9f96e118f6cdf0dee21d7764afb0c8270b6b/apps/codeforafrica/src/components/useFilterQuery/useFilterQuery.js#L17-L19) hook to take the whole tag, rather than just the slug. That is why currently in production you can't filter stories & opportunities by Tag.